### DISCUSSION TITLE
List Venmo Credit Card Transactions

### DISCUSSION THREAD

**Discussion by marwan-at-work** [mmohades/Venmo#69] [Q&A]
Hi there,
Thank you for putting together the API documentation. I was wondering if there's a way to find out how you can list venmo's credit card transactions. 

The `/stories/target-or-actor` shows venmo payments, but in the UI you go to a different section to view your credit card transactions and I'm not sure how to find out the API endpoint for it. 

Thanks!

**Comment by marwan-at-work** [comment]
Update: did some packet sniffing and it looks like the `credit-card/transactions` is a potential answer here.

**Reply by mmohades** [reply]
Nice! How does the response look like?
I don't own a Venmo credit card so I don't know how it works.

**Reply by marwan-at-work** [reply]
@mmohades it's pretty much the reason I got the credit card so I can use the API to go over my transactions. I am unaware of any US-based credit card that has an official API one can use for personal purposes.

The response looks like this: 

```
{
    "pagination": {
        "next": null
    },
    "data": [
        {
            "merchant": {
                "category": {
                    "image_url": "category-thumbnail",
                    "name": "category-name"
                },
                "name": "merchant-name"
            },
            "reference_number": null,
            "description": "merchant-name",
            "created_at": "2022-04-05",
            "transaction_type": "purchase",
            "amount": 2311, // in cents (multiply by *0.01 to get amount in dollars)
            "status": "pending",
            "scheduled_date": null,
            "id": "123123123",
            "detail_url": "/v1/credit-card/transactions/123123123",
            "paid_by": null
        }
    ]
}
```

**Reply by mmohades** [reply]
Yeah it can be very helpful. I'll try to add it to the wrapper at some point. Need to find someone that uses the credit card and allows me to do a mitm proxy on their phone 😄
